src/component/toolbox/feature/SaveAsImage.ts

```diff
@@ -73,15 +73,42 @@ class SaveAsImage extends ToolboxFeature<ToolboxSaveAsImageFeatureOption> {
         }
         // IE
         else {
-            if (window.navigator.msSaveOrOpenBlob) {
-                const bstr = atob(url.split(',')[1]);
-                let n = bstr.length;
-                const u8arr = new Uint8Array(n);
-                while (n--) {
-                    u8arr[n] = bstr.charCodeAt(n);
+            if (window.navigator.msSaveOrOpenBlob || isSvg) {
+                const parts = url.split(',');
+                // data:[<mime type>][;charset=<charset>][;base64],<encoded data>
+                // see https://css-tricks.com/data-uris/
```

Comment:
It's better to avoid URL referencing like this. It may have license issues

Reply:
Thanks for your reminder. Will remove it.